psyneulink/library/components/mechanisms/processing/integrator/ddm.py

```diff
@@ -1164,14 +1164,17 @@ def _execute(
                 return_value[self.DECISION_VARIABLE_INDEX] = threshold
             return return_value
 
-    def reinitialize(self, *args, execution_context=None):
+    def reinitialize(self, *args, execution_context=NotImplemented):
```

Comment:
If using most recent execution id is the way we want to go with no-arg reinitialize, we should do it in a way that handles all the reinitialize methods in its own commit/pr